.gitignore

```diff
@@ -178,3 +178,5 @@ $RECYCLE.BIN/
 
 # Visual Studio Code
 .vscode/
+/errors.txt
```

Comment:
Artefact from my testing. I have small executable in the root of the roslyn folder that helps me extract the all of unit-test failures into one text document. Which makes it easy to use from the command window, I use during build and test.